### ISSUE TITLE
Insertion Library: Fix menu focus when using keyboard

### ISSUE THREAD

**Issue by miina** [swissspidy/web-stories-wp#10607]
## Bug Description
Currently, when closing the "More" menu, the focus is lost "somewhere" and doesn't return to the original element/button.

Additionally, when tabbing inside a menu and the menu is closed, the button for opening the menu keeps `tabIndex` 0, meaning that `Shift+Tab` will move the focus into the button with navigation with arrow keys not being possible 

## Expected Behaviour
The focus should return to the button after closing the menu

## Steps to Reproduce
1. Tab into Media
2. Open the "More" menu
3. Click "Esc"
4. See the focus is not back on the media element (or button, after #10388)
5. Note: the same goes for all menus once #10388 has been merged.

Tabbing:
1. Move keyboard focus to local media item
2. Press enter to open insertion menu
3. Press tab to move focus to canvas
4. Press shift-tab to move focus back to media item, and observe focus moves to insertion menu button
5. Try to navigate to other media items using arrow keys and observe it does not work
6. Press shift-tab again to move focus back to the entire media item
7. Try to navigate to other media items using arrow keys and observe it does work

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

**Comment by swissspidy** [comment]
Related: #9913

**Comment by felipebochehin87** [comment]
Tested against https://stories-qa-wordpress-amp.pantheonsite.io/wp-admin/, using PR #10780.

1. Hit tab until the focus reach + button
2. Hit space
3. Hit Esc - focus returns to the + button
4. Repeated the same steps for the ... button
5. When cancelling a pop-up message, focus returned to the button from where it came from (...)
6. Compared the main (with the issue) with the PR (fixed)


https://user-images.githubusercontent.com/96055551/156779052-7d200012-0ea8-4cd9-8d23-082d00fbf1d8.mp4
